pkg/auth/inject.go

```diff
@@ -53,13 +54,15 @@ func (m DependencyMap) Provide(dependencyName string, r *http.Request) interface
 		}
 	case "PasswordAuthProvider":
 		tConfig := config.GetTenantConfig(r)
+		maskFormatter := logging.CreateMaskFormatter(sensitiveLoggerValues(r), &logrus.TextFormatter{})
 		return password.NewProvider(
 			db.NewSQLBuilder("auth", tConfig.AppName),
 			db.NewSQLExecutor(r.Context(), db.NewContextWithContext(r.Context(), openDB(tConfig))),
-			logging.CreateLogger(r, "provider_password"),
+			logging.CreateLogger(r, "provider_password", maskFormatter),
 		)
 	case "HandlerLogger":
-		return logging.CreateLogger(r, "handler")
+		maskFormatter := logging.CreateMaskFormatter(sensitiveLoggerValues(r), &logrus.TextFormatter{})
+		return logging.CreateLogger(r, "handler", maskFormatter)
```

Comment:
I pass `maskFormatter` as parameter due to not sure I think we may want to configure formatter in some cases. For example, we may want to use `JSONFormatter` in some module in stead of `TextFormatter`. 

And second, I think `logger` module should work without knowing application-level logic.

Reply:
I think the logger module should work without knowing the gear specific logic.

Gear is our application. So if a logic is univerial enough that will apply too all skygear gear, it can be the logic in the core lib.

So the maskFormatter fall into this category, and it can be in core lib.

The discussion should focus on “are different mask is required for different gear and different tanent?”
My answer is yes, we may allow the tanent to configure log to remote logger/without the mastk.